src/platform/macos/display.mm

```diff
@@ -142,18 +142,22 @@
 
     auto display = std::make_shared<av_display_t>();
 
+    // Default to main display
     display->display_id = CGMainDisplayID();
-    if (!display_name.empty()) {
-      auto display_array = [AVVideo displayNames];
-
-      for (NSDictionary *item in display_array) {
-        NSString *name = item[@"name"];
-        if (name.UTF8String == display_name) {
-          NSNumber *display_id = item[@"id"];
-          display->display_id = [display_id unsignedIntValue];
-        }
+
+    // Print all displays available with it's name and id
+    auto display_array = [AVVideo displayNames];
+    BOOST_LOG(info) << "Detecting displays..."sv;
+    for (NSDictionary *item in display_array) {
+      NSNumber *display_id = item[@"id"];
+      // We need show display's product name and corresponding display number given by user
+      NSString *name = item[@"displayName"];
+      BOOST_LOG(info) << "Detected display: "sv << name.UTF8String << " (id: "sv << [NSString stringWithFormat:@"%@", display_id].UTF8String << ")"sv;
```

Comment:
Can you update the Linux format to more closely match what you did here? If so, I think there is an opportunity to commonize some things, such as:

- Using the term "display" instead of "monitor"
- A combined section in the config UI for Linux and macOS
- Common localization strings